compiler/optimizer/LoopReducer.cpp

```diff
@@ -1694,7 +1685,7 @@ TR_Arraytranslate::checkLoad(TR::Node * loadNode)
 bool
 TR_Arraytranslate::checkStore(TR::Node * storeNode)
    {
-   if (storeNode->getOpCodeValue() != TR::cstorei && storeNode->getOpCodeValue() != TR::bstorei)
```

Comment:
This also doesn't look right - I think this needs to change to sstorei with appropriate compensation.

Reply:
The mistake was fixed in bd43e4f